js/stringProcessing/getSentences.js

```diff
@@ -50,7 +53,7 @@ var invalidateOnCapital = function( text, positions, i ) {
 var filterPositions = function( text, positions ) {
 	return filter( positions, function( position, index ) {
 		if ( !isUndefined( positions[ index + 1 ] ) ) {
-			if ( invalidateOnWhiteSpace( text, positions[ index ] ) || invalidateOnCapital( text, positions, index ) ) {
+			if ( invalidateOnCharacter( text, positions[ index ] ) || invalidateOnCapital( text, positions, index ) ) {
```

Comment:
Altering the linke above this one to `return;` when undefined will allow you to stick this if-statement underneath it and remove an extra level of (unnecessary) indentation.